release-notes/opensearch-release-notes-2.4.0.md

```diff
@@ -0,0 +1,704 @@
+# OpenSearch and Dashboards 2.4.0 Release Notes
+
+## Release Highlights
+
+OpenSearch 2.4.0 is the first release with OpenSearch Security Analytics & Security Analytics Dashboards.
+Security Analytics consist of two plugins, security-analytics backend plugin for OpenSearch, and a securityAnalyticsDashboards frontend plugin for OpenSearch Dashboards.
+The Security Analytics Dashboards plugin lets you manage your Security Analytics plugin to generate critical security insights from their existing security event logs (such as firewall logs, windows logs, authentication audit logs, etc.) directly from OpenSearch Dashboards.
+
+## Release Details
+
+[OpenSearch and OpenSearch Dashboards 2.4.0](https://opensearch.org/versions/opensearch-2-4-0.html) includes the following features, experimental features, enhancements, bug fixes, infrastructure, documentation, maintenance, and refactoring updates.
```

Comment:
This hyperlink to direct users to the artifacts download page is suggested in https://github.com/opensearch-project/opensearch-build/issues/2707